java-desktop/src/main/java/com/couchbase/todo/controller/MainController.java

```diff
@@ -40,5 +53,18 @@ public void initialize(URL location, ResourceBundle resources) {
             TodoApp.goToPage(this.stage, TodoApp.LOGIN_FXML);
         });
         configMenuItem.setOnAction(event -> TodoApp.goToPage(this.stage, TodoApp.CONFIG_FXML));
+
+        //default select enable JSONResults
+        group.selectToggle(onToggle);
+
+        group.selectedToggleProperty().addListener(new ChangeListener<Toggle>() {
+            @Override
+            public void changed(
+                ObservableValue<? extends Toggle> observable, Toggle oldValue, Toggle newValue) {
+                Toggle t = group.getSelectedToggle();
+                if (t == onToggle) {jsonBoolean.set(true);}
+                else {jsonBoolean.set(false);}
+            }
```

Comment:
I think this could be better said:
jsonBoolean.set(group.getSelectedToggle() == onToggle)